npc/custom/warper.txt

```diff
@@ -424,47 +451,72 @@ G6: Go("schg_dun01",200,124);
-    	"Wolfchev's Laboratory",I12;
+menu	"Bakonawa Lake",I1, "Bangungot Hospital 2F",I2, "Buwaya Cave",I3, "Eclage Interior",I4,
+    	"Endless Tower",I5, "Faceworms Nest",I6, "Geffen Magic Tournament",I7, "Hazy Forest",I8, 
+		"Horror Toy Factory",I9, "Malangdo Culvert",I10, "Nidhoggur's Nest",I11, "Octopus Cave",I12,
+    	"Old Glast Heim",I13, "Orc's Memory",I14, "Sarah and Fenrir",I15, "Sara Memory",I16,
+		"Sealed Shrine",I17, "Wolfchev's Laboratory",I18;
 
 I1: Restrict("RE");
-	Go("ma_scene01",172,175);
+	Go("ma_scene01",172,175,1);
 I2: Restrict("RE");
-	Go("ma_dun01",151,8);
+	Go("ma_dun01",151,8,1);
 I3: Restrict("RE");
-	Go("ma_fild02",316,317);
-I4: Go("e_tower",72,112);
-I5: Restrict("RE");
-	Go("bif_fild01",161,334);
+	Go("ma_fild02",316,317,1);
+I4: Restrict("RE");
+	Go("ecl_hub01",129,12,1);
+I5: Go("e_tower",72,112,1);
 I6: Restrict("RE");
-	Go("mal_in01",164,21);
-I7: Go("nyd_dun02",95,193);
+	Go("dali",85,64,1);
+I7: Restrict("RE");
+	Go("dali",94,141,1);
 I8: Restrict("RE");
-	Go("mal_dun01",152,230);
+	Go("bif_fild01",161,334,1);
 I9: Restrict("RE");
-	Go("glast_01",204,268);
-I10: Go("gef_fild10",240,198);
-I11: Go("monk_test",306,143);
+	Go("xmas",234,298,1);
+I10: Restrict("RE");
+	Go("mal_in01",164,21,1);
+I11: Go("nyd_dun02",95,193,1);
 I12: Restrict("RE");
-	Go("lhz_dun04",148,269);
+	Go("mal_dun01",152,230,1);
+I13: Restrict("RE");
+	Go("glast_01",204,268,1);
+I14: Go("gef_fild10",240,198,1);
+I15: Restrict("RE");
+	Go("dali",92,141,1);
+I16: Restrict("RE");
+	Go("dali",133,108,1);
+I17: Go("monk_test",306,143,1);
+I18: Restrict("RE");
+	Go("lhz_dun04",148,269,1);
 
 // --------------------------------------------------
 	Special:
 // --------------------------------------------------
-menu	"Auction Hall",S1, "Battlegrounds",S2, "Casino",S3, "Eden Group Headquarters",S4,
-    	"Gonryun Arena",S5, "Izlude Arena",S6, "Monster Race Arena",S7, "Turbo Track",S8;
+menu	"Auction Hall",S1, "Battlegrounds",S2, "Casino",S3, "Dimensional Rift",S4,
+		"Eden Group Headquarters",S5, "Gonryun Arena",S6, "Izlude Arena",S7, 
+		"Monster Race Arena",S8, "Turbo Track",S9;
 
-S1: Go("auction_01",22,68);
-S2: Go("bat_room",154,150);
-S3: Go("cmd_in02",179,129);
+S1: Go("auction_01",22,68,1);
+S2: Go("bat_room",154,150,1);
+S3: Go("cmd_in02",179,129,1);
 S4: Restrict("RE");
-	Go("moc_para01",31,14);
-S5: Go("gon_test",48,10);
-S6: Go("arena_room",100,88);
-S7: Go("p_track01",62,41);
-S8: Go("turbo_room",99,114);
+	Go("dali",113,82,1);
+S5: Restrict("RE");
+	Go("moc_para01",31,14,1);
+S6: Go("gon_test",48,10,1);
+S7: Go("arena_room",100,88,1);
+S8: Go("p_track01",62,41,1);
+S9: Go("turbo_room",99,114,1);
+
+
+OnInit:
+	.Satan_Morroc = 1;	//	0 will enable moc_fild 4,5,6,8,9,10,14,15 wjile disable moc_fild 20,21,22 Default is 1
+	//Warp the player to random Coordinates in the map .
+	setarray .random_coordinates$,"";	//	Example: setarray .random$,"prontera","Alberta",Aldebaran";
```

Comment:
I'm against this setting. Using 0 for warps coordinates is more productive.
`Go("payon",0,0);` instead of `Go("payon",179,100);` for random warp on payon.